Tests/L0/ServiceFabricDeploy/AadDeploy.ps1

```diff
@@ -21,6 +21,7 @@ Register-Mock Get-VstsInput { $publishProfilePath } -- -Name publishProfilePath
 Register-Mock Get-VstsInput { $applicationPackagePath } -- -Name applicationPackagePath -Require
 Register-Mock Get-VstsInput { $serviceConnectionName } -- -Name serviceConnectionName -Require
 Register-Mock Get-VstsInput { "false" } -- -Name compressPackage
+Register-Mock Get-VstsInput { "SameAppTypeAndVersion" } -- -Name overwriteBehavior
```

Comment:
Same for the other tests

---
In reply to: [97853869](https://github.com/Microsoft/vsts-tasks/pull/3382#discussion_r97853869) [](ancestors = 97853869)